travis/setup_dependencies_common.sh

```diff
@@ -9,6 +9,10 @@ conda config --add channels defaults
 
 shopt -s nocasematch
 
+LATEST_ASTROPY_STABLE=1.1
+LATEST_ASTROPY_LTS=1.0
+LATEST_NUMPY_STABLE=1.11
```

Comment:
I don't think things will work well with Numpy, because if you update just numpy from 1.5 to 1.11, I think a lot of packages built on Numpy will fail. I really hope the conda build gets fixed soon.